build.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<project basedir="." default="build" name="WebApp">
+    <property name="gwt.args" value="" />
+    <property location="JavaUniverse-JS/libs" name="gwt.sdk" />
+    <property location="JavaUniverse/libs" name="JavaUniverse.sdk" />
+
+    <path id="project.class.path">
+        <pathelement location="${JavaUniverse.sdk}/j2objc-annotations-1.3.jar" />
+        <pathelement location="JavaUniverse/src/main/java" />
```

Comment:
I believe there are many duplications, why do we need add both Java source file and Java class file?